Source/Core/Core/PowerPC/SignatureDB/SignatureDB.h

```diff
@@ -18,6 +18,51 @@ class SignatureDBFormatHandler;
 class SignatureDB
 {
 public:
+  enum class Handler
+  {
+    DSY,
+    CSV
+  };
+  SignatureDB(Handler handler = Handler::DSY);
```

Comment:
Yeah, I know but that's actually the Dolphin preferred way to handle signatures. I also assume it wasn't designed to handle something else than hash signature database in the first place. Basically Dolphin uses the totaldb.dsy database most of the time when dealing with symbol generation such as SymbolMap.

I can remove that default constructor if you want, I'll just have to change other parts of the code to reflect that.